packages/jmix-react-antd/src/ui/EntityHierarchyTree/EntityHierachyTree.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import {Tree, TreeProps} from "antd";
+import {DataNode} from "rc-tree/lib/interface";
+
+export type HierarchyTreeNode = Omit<DataNode, "children" | "key"> & {
+  id: string,
+  _instanceName: string,
+  [child: string]: string,
```

Comment:
Why do we need this prop?

Reply:
Prop " [child: string]" that refer to parent id (should be renamed) can have any name defined by `hierarchyProperty` prop, i.e. `hierarchyProperty="parentTask"`

from task:
> ... hiearachyProperty prop which determines which of the entity's properties is the link to parent entity instance.